google-cloud-bigtable/google-cloud-bigtable.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = ">= 2.0.0"
 
-  gem.add_dependency "google-gax", "~> 1.0"
+  gem.add_dependency "google-gax", "~> 1.2.0"
```

Comment:
I believe the dependency should remain at 1.0. The problem with the tests mocking at the implementation level should not prohibit users from choosing to run an older, API compatible version of google-gax.

Reply:
Got it. changed `google-gax` version to 1.0